src/sticky_index.rs

```diff
@@ -0,0 +1,13 @@
+use pyo3::prelude::*;
+use yrs::StickyIndex;
+
+#[pyclass]
+pub struct PyStickyIndex {
```

Comment:
I'd prefer to call it `StickyIndex` instead of `PyStickIndex` to keep it consistent with the naming convention used for other objects.